sim/pokemon.ts

```diff
@@ -382,7 +382,7 @@ export class Pokemon {
 		this.m = {};
 	}
 
-	get moves() {
+	get moves(): ReadonlyArray<string> {
```

Comment:
```suggestion
	get moves(): readonly string[] {
```

Reply:
With this change I get tons of errors, starting with:

ERROR: sim/pokemon.ts:385:23 - block is empty

Am I doing something wrong?

Reply:
I don't know that Zarel's suggestion is valid pre Typescript 3.4 (https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html). You probably need to update your Typescript version.

Reply:
I thought we already updated our TypeScript version?

Reply:
We did, I dont know if @urkerab did (is it automatic? if the repo updates `package.json`, will running `npm test` cause it to download the new deps?)

Reply:
Oh, it's not automatic, yeah, you'll all need to manually rerun `npm i`.

Reply:
(I had to downgrade `node` so that I could upgrade `npm` and then re-upgrade `node` so I could `npm install` and `npm test` but it all works now, phew!)